iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -2057,4 +2057,26 @@ public void testSnycProperties() throws TException, InterruptedException {
   private String getCurrentSnapshotForHiveCatalogTable(org.apache.iceberg.Table icebergTable) {
     return ((BaseMetastoreTableOperations) ((BaseTable) icebergTable).operations()).currentMetadataLocation();
   }
+
+  @Test
+  public void testCreateTableWithPercentInName() throws TException, IOException, InterruptedException {
```

Comment:
Could you please remove the unnecessary throws?

Reply:
BTW, as this issue is only related hive catalog. you can add this code line:
`Assume.assumeTrue("This test is only for hive catalog", testTableType == TestTables.TestTableType.HIVE_CATALOG);`

Reply:
Made both changes